python/mujoco/__init__.py

```diff
@@ -54,7 +54,15 @@
 def _load_all_bundled_plugins():
   for directory, _, filenames in os.walk(PLUGINS_DIR):
     for filename in filenames:
-      PLUGIN_HANDLES.append(ctypes.CDLL(os.path.join(directory, filename)))
+        if os.path.splitext(filename)[-1] == ".so":
```

Comment:
This need to be platform-dependent. See https://github.com/deepmind/mujoco/blob/main/python/setup.py#L73

Reply:
Fixed. I added the file extensions `.dll`, `.dylib` & `.so`. Are you fine with just matching the extension or do you want me to match the complete filename as in the linked `setup.py`?